docs/src/pages/index.js

```diff
@@ -9,18 +9,68 @@ import styles from './index.module.css';
 
 function HomepageHeader() {
   const {siteConfig} = useDocusaurusContext();
+
+  const dockerScript = "docker run -d -t -i \\\n" +
+      "-e KLAW_CLUSTERAPI_ACCESS_BASE64_SECRET=\"dGhpcyBpcyBhIHNlY3JldCB0byBhY2Nlc3MgY2x1c3RlcmFwaQ==\" \\\n" +
+      "-p 9343:9343 \\\n" +
+      "--name klaw-cluster-api aivenoy/klaw-cluster-api:nightly --add-host host.docker.internal:host-gateway \\\n" +
+      "&& docker run -d -t -i \\\n" +
+      "-e KLAW_CLUSTERAPI_ACCESS_BASE64_SECRET=\"dGhpcyBpcyBhIHNlY3JldCB0byBhY2Nlc3MgY2x1c3RlcmFwaQ==\" \\\n" +
+      "-e KLAW_QUICKSTART_ENABLED=true \\\n" +
+      "-e SPRING_DATASOURCE_URL=\"jdbc:h2:file:/klaw/klawprodb;DB_CLOSE_ON_EXIT=FALSE;DB_CLOSE_DELAY=-1;MODE=MySQL;CASE_INSENSITIVE_IDENTIFIERS=TRUE;\" \\\n" +
+      "-p 9097:9097 --name klaw-core aivenoy/klaw-core:nightly \\\n" +
+      "&& docker run -d -t -i -p 2181:2181 --add-host host.docker.internal:host-gateway \\\n" +
+      "-e ALLOW_ANONYMOUS_LOGIN=yes --name klaw-zookeeper bitnami/zookeeper:3.8 \\\n" +
+      "&& docker run -d -t -i -p 9092:9092 --add-host host.docker.internal:host-gateway \\\n" +
+      "-e KAFKA_CFG_ZOOKEEPER_CONNECT=host.docker.internal:2181 -e ALLOW_PLAINTEXT_LISTENER=yes \\\n" +
+      "-e KAFKA_ADVERTISED_LISTENERS=PLAINTEXT://host.docker.internal:9092 \\\n" +
+      "--name klaw-kafka bitnami/kafka:3.3 \\\n" +
+      "&& docker run -d -t -i -p 8081:8081 --add-host host.docker.internal:host-gateway \\\n" +
+      "-e SCHEMA_REGISTRY_KAFKA_BROKERS=PLAINTEXT://host.docker.internal:9092 \\\n" +
+      "--name klaw-schema-registry bitnami/schema-registry:latest";
+
   return (
     <header className={clsx('hero hero--primary', styles.heroBanner)}>
       <div className="container">
-        <h1 className="hero__title">{siteConfig.title}</h1>
-        <p className="hero__subtitle">{siteConfig.tagline}</p>
-        <div className={styles.buttons}>
-          <Link
-            className="button button--secondary button--lg"
-            to="/docs/getstarted">
-            Get started
-          </Link>
-        </div>
+          <div className="row">
+              <div className="col col--4">
+                  <h1 className="hero__title">{siteConfig.title}</h1>
+                  <p className="hero__subtitle">{siteConfig.tagline}</p>
+                  <div className={styles.buttons}>
+                      <Link
+                          className="button button--secondary button--lg"
+                          to="/docs/getstarted">
+                          Go to docs
+                      </Link>
+                  </div>
+              </div>
+              <div className="col col--4">
+                  <h4>Quick start : trying Klaw is as easy as ...</h4>
```

Comment:
Quick start : trying Klaw is as easy as ...""

Quick start: Try klaw out in two minutes or less!